### DISCUSSION TITLE
install toltec for RM2

### DISCUSSION THREAD

**Discussion by Sainthilaire-thomas** [toltec-dev/toltec#307] [Support] [answered]
Hello, when i try to install toltec on RM2, it says that the cheksum is not correct, i have used the recent key on toltec site, may be i should uninstall something to make a clean start ?  or is this related to RM2 (I have seen a similar problem with RM1 but solved with the correct key) Thank you very much if you have an idea, here is screenshot:
![Capture d’écran (94)](https://user-images.githubusercontent.com/64955536/110424400-08c3b400-80a3-11eb-9aca-5f0ca88c2499.png)

**Comment by danshick** [comment, marked as answer]
It appears you have a right angle bracket seven characters in to the sha hash. I would suspect that's your issue. Maybe a bad copy paste? I'd head back over to the github page or toltec website and copy it again.

**Reply by Sainthilaire-thomas** [reply]
thank you very much danshick, may be because i have typed at hand because paste is not working on my remarkable command line, i will find a workaround to test again and keep you posted

**Reply by Sainthilaire-thomas** [reply]
ok it works, (I am new i did not know how to copy acharacters in ssh command line) toltec is installed, thank you very much for your kind help, have a nice day

**Reply by danshick** [reply]
No problem.
